spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+describe 'UsersControllers', type: :request do
+  describe 'GET #index' do
+    it 'returns a successful response' do
+      get '/users'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      get '/users'
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get '/users'
+      expect(response.body).to include('Tom')
+    end
+  end
+
+  describe 'GET #show' do
+    let(:user) { User.create(name: 'Ruby Guy') }
+
+    it 'returns a successful response' do
+      get "/users/#{user.id}"
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the show template' do
+      get "/users/#{user.id}"
+      expect(response).to render_template(:show)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get "/users/#{user.id}"
+      expect(response.body).to include('Ruby Guy')
+    end
+  end
```

Comment:
![Screenshot_148](https://github.com/kifle23/blog-app/assets/69638013/8337e7d6-008c-496f-a8ca-ef84be2afa9f)

- [ ]  Please ensure that the tests written for the user and show actions for users controller are passing

Reply:
Hello @prolajumokeoni, if you don't mind, may I ask what specifically I should do; because on my end all tests are passing.
![image](https://github.com/kifle23/blog-app/assets/68891198/e08c98bd-8ddf-4874-b457-aead9fb141aa)